.gitignore

```diff
@@ -3,6 +3,7 @@
 /docs
 /node_modules
 /config/travis/deploy.key
-.*
 *.diff.png
 *.new.png
+.DS_Store
+/.jest
```

Comment:
Why not ignore all files and folders starting with dot?

Reply:
Cos we want the .vscode/ folder? Looking at YamJS' we manually list them

Reply:
`git add .vscode -f`, and keep the `.gitignore` as it was.

Reply:
Things like `.DS_Store` shouldn't live in a project's `.gitignore`. They should live in each person's global `.gitignore`. YamJS is not a good guide for anything.